src/markdown-pages/install-with-kurl/cis-compliance.md

```diff
@@ -0,0 +1,48 @@
+---
+path: "/docs/install-with-kurl/cis-compliance"
+date: "2021-12-14"
+weight: 24
+linktitle: "CIS Compliance"
+title: "CIS Compliance"
+isAlpha: false
+---
+The kURL installer can be configured to be Center for Internet Security (CIS) compliant. This is an opt-in feature that is configured in the kURL specification by setting the `kurl.cisCompliance` field to `true`.
```

Comment:
@jala-dx @emosbaugh can we put more details here? I had thought we would outline all of the settings that were behind the flag.

Reply:
@JustinN-Replicated Just committed these changes, though I'm not sure why the preview isn't rebuilding.